spec/requests/api/docomo/authorization_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+require 'securerandom'
+
+describe 'Docomo API behavior' do
```

Comment:
こちらのPR (https://github.com/quipper/sbpayment.rb/pull/140)をマージした関係で各specのtoplevelで `RSpec.describe` としてくださいmm

```suggestion
RSpec.describe 'Docomo API behavior' do
```